src/ripple/app/paths/impl/Steps.h

```diff
@@ -222,62 +271,73 @@ bool operator==(Strand const& lhs, Strand const& rhs)
             return false;
     return true;
 }
+/// @endcond
 
-/*
-   Create a strand for the specified path
+/**
+   Create a Strand for the specified path
 
    @param sb view for trust lines, balances, and attributes like auth and freeze
    @param src Account that is sending assets
    @param dst Account that is receiving assets
    @param deliver Asset the dst account will receive
    (if issuer of deliver == dst, then accept any issuer)
-   @param sendMax Optional asset to send.
+   @param limitQuality Do not use liquidity below this quality threshold
```

Comment:
The `limitQuality` parameter on this function is only used for offer crossing, and only for book steps. The description should be re-written to describe what this actually does or it's going to confuse people.